### PR TITLE
Update the Spanish language

### PR DESCRIPTION
**Pull request by CarlBarr**

### CODE REVIEW

**Comment by Ekdohibs** [comment]
@kaeza could you please review? It seems that you and @CarlBarr  don't agree on the spelling on some words (could this be related to how Spanish is spelled in different places? If so, do you think we should make several Spanish versions?).

File: locale/es.txt [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
-Medium Bag = Bolsa Mediana
-Large Bag = Bolsa Grande
+Bags = Bolsos
+Bag @1 = Bolso @1
+Bag 1 = Bolso 1
+Bag 2 = Bolso 2
+Bag 3 = Bolso 3
+Bag 4 = Bolso 4
+Small Bag = Bolso Pequeña
+Medium Bag = Bolso Mediana
```
Comment (kaeza) [inline locale/es.txt]: Pequeñ**o** y Median**o**?

File: locale/es.txt [hunk 2]
```diff
@@ -44,29 +54,51 @@ cooking = hornear
 alloy cooking = horneado de aleación
 Copy to craft grid: = Copiar al cuadro de elaboración
 All = Todos
-Recipe %s of %s = Receta %s de %s
 Alternate = Alternar
-Crafting Grid =  
+Crafting Grid =  Construir objetos
+Show next recipe = Mostrar la siguiente receta
+Show next usage = Mostrar el siguiente uso
+Show previous recipe = Mostrar la resera anterior
```
Comment (kaeza) [inline locale/es.txt]: Receta?

File: locale/es.txt [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
 Page = Página
 %s of %s = %s de %s
 Filter = Filtro
-Search = 
+Search = Buscar
+Reset search and display everything = Limpiar la busqueda y mostrar todo
 
 ### register.lua ###
 Can use the creative inventory = Puede usar el inventario creativo
+Forces Unified Inventory to be displayed in Full mode if Lite mode is configured globally =
```
Comment (kaeza) [inline locale/es.txt]: Untranslated string.

File: locale/es.txt [hunk 4]
```diff
@@ -44,29 +54,51 @@ cooking = hornear
...
-Recipe %s of %s = Receta %s de %s
 Alternate = Alternar
-Crafting Grid =  
+Crafting Grid =  Construir objetos
+Show next recipe = Mostrar la siguiente receta
+Show next usage = Mostrar el siguiente uso
+Show previous recipe = Mostrar la resera anterior
+Show previous usage = Mostrar el uso anterior
+# Shown for huge crafting recipes; try to keep the line length short and use multiple line breaks as needed
+This recipe is too\nlarge to be displayed. = Esta reseta es\ndemaciado grande\npara que se muestre
```
Comment (kaeza) [inline locale/es.txt]: Re**c**eta.

File: locale/es.txt [hunk 5]
```diff
@@ -44,29 +54,51 @@ cooking = hornear
...
+# %s = group name (e.g. wool)
+Any item belonging to the %s group = Cualquier elemento perteneciente al grupo de %s
+# %s = List of “and”-concatenated group names
+Any item belonging to the groups %s = Cualquier elemento que pertenezca a los grupos %s
+Recipe %d of %d = Receta %d de %d
+Usage %d of %d = Uso %d de %d
+No recipes = No hay resetas
```
Comment (kaeza) [inline locale/es.txt]: Re**c**etas.

**Comment by CarlBarr** [comment]
I corrected the errors at https://github.com/minetest-mods/unified_inventory/pull/109

**Pull request closed without merging**